### PR TITLE
Test dataset import

### PR DESCRIPTION
**Pull request by dominikl**
This PR prepares all necessary files for the import of the two sample datasets.

### CODE REVIEW

**Comment by sbesson** [comment]
Closing/reopening to run Travis.

**Comment by dominikl** [comment]
I think the PR is ready to review now @sbesson . The "scripts" are likely to change again until we have agreed on the final version of the assays.txt layout and what information will get into there (e.g. ICD terms/ids, etc.), that'll be a matter of a separate PR. I think in the end we'd ideally have one script which you simply launch and it'll generate the filepaths.tsv and annotation.csv without any manual intervention.

File: scripts/annotations/PrefixColumns.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        String concat = "Dataset:name:";
+        
+        // the column index
+        int index = 0;
+        
+        char separator = '\t';
+        
+        // END Parameters
+        
+        BufferedReader r = new BufferedReader(new FileReader(in));
```
Comment (jburel) [inline scripts/annotations/PrefixColumns.java]: Maybe user try-with-resources

**Comment by sbesson** [comment]
Tested a few times the raw image import by running `bash parallel_image_import.sh` after deleting the top-level container. Overall the logic works as expeected. A couple of comments:

- the first run led into some concurrency issues of type `ResourceError`. A short `--delay` between each run might be a workaround. @dominikl also suggested we could make more use of the GNU parallel `--retry-failed` option
- results are currently stored under `rslt/1/<very_long_name_with_the_complete_import_options>` which is not extremely friendly :) To improve this, I suspect the only viable way would be to wrap each import command into a single shell script e.g.

  ```
  commands.txt  # lists 0001.sh, 0002.sh...
  0001.sh
  0002.sh
  ...
  0980.sh
  ```

   This should result in a more manageable `rslt/1/{0001,0002,...}.sh/` tree
- with 10 threads, individual images import in ~10-12s so the 2 datasets takes 15-20min to import in total.

Once the annotation is successfully testing, I propose to merge and have improvements in follow-up PRs.

File: scripts/annotations/PrefixColumns.java (hunk 1 above)
Comment (dominikl) [reply scripts/annotations/PrefixColumns.java]: 👍 I have to do a general overhaul of the scripts, they are still pretty "quick and dirty". As there will be many HPA projects coming in (in hopefully exactly the same format),  ideal would be to finally have one script which generates all necessary files (filePaths.tsv, annotation.csv, etc.) in one go.

**Pull request merged**